docker-compose.yml

```diff
@@ -0,0 +1,56 @@
+version: '3.8'
+services:
+  app: 
+    container_name: messaging_backend
+    build: .
+    ports:
+      - 8080:8080
+    restart: on-failure
+    volumes: 
+      - api:/urs/src/app
+    depends_on:
+      - fullstack-mysql 
+    networks:
+      - fullstack
+
+  fullstack-mysql:
+    image: mysql:5.7
+    container_name: messaging_backend_mysql
+    ports:
+      - 3306:3306
+    environment:
```

Comment:
@binishmaharjan 
Change this to read the .env file like this
```
 env_file:
      - .env
```

Reply:
@diskii 
sorry can you explain this abit more 😅

Reply:
@
Change this
```
environment:
      - MYSQL_ROOT_HOST=${DB_HOST}
      - MYSQL_USER=${DB_USER}
      - MYSQL_PASSWORD=${DB_PASSWORD}
      - MYSQL_DATABASE=${DB_NAME}
      - MYSQL_ROOT_PASSWORD=${DB_PASSWORD} 
```
 to 
```
 env_file:
      - .env
```